Sources/MapboxMaps/Foundation/BaseMapView.swift

```diff
@@ -52,87 +66,57 @@ open class BaseMapView: UIView, CameraViewDelegate {
     public var centerCoordinate: CLLocationCoordinate2D {
-        get {
-            guard let center = cameraOptions.center else {
-                fatalError("Center is nil in camera options")
-            }
-            return center
-        } set {
-            cameraView.centerCoordinate = newValue
+        guard let center = cameraOptions.center else {
+            fatalError("Center is nil in camera options")
         }
+        return center
     }
 
-    /// The map's  zoom level.
+    /// The map's current zoom level.
     public var zoom: CGFloat {
-        get {
-            guard let zoom = cameraOptions.zoom else {
-                fatalError("Zoom is nil in camera options")
-            }
-            return CGFloat(zoom)
-        } set {
-            cameraView.zoom = newValue
+        guard let zoom = cameraOptions.zoom else {
+            fatalError("Zoom is nil in camera options")
         }
+        return CGFloat(zoom)
     }
 
-    /// The map's bearing, measured clockwise from 0° north.
+    /// The map's current bearing, measured clockwise from 0° north.
     public var bearing: CLLocationDirection {
-        get {
-            guard let bearing = cameraOptions.bearing else {
-                fatalError("Bearing is nil in camera options")
-            }
-            return CLLocationDirection(bearing)
-        } set {
-            cameraView.bearing = CGFloat(newValue)
+        guard let bearing = cameraOptions.bearing else {
+            fatalError("Bearing is nil in camera options")
         }
+        return CLLocationDirection(bearing)
     }
 
-    /// The map's pitch, falling within a range of 0 to 60.
+    /// The map's current pitch, falling within a range of 0 to 60.
     public var pitch: CGFloat {
-        get {
-            guard let pitch = cameraOptions.pitch else {
-                fatalError("Pitch is nil in camera options")
-            }
-
-            return pitch
-        } set {
-            cameraView.pitch = newValue
-        }
-    }
-
-    /// The map's camera padding
-    public var padding: UIEdgeInsets {
-        get {
-            return cameraOptions.padding ?? .zero
-        } set {
-            cameraView.padding = newValue
+        guard let pitch = cameraOptions.pitch else {
+            fatalError("Pitch is nil in camera options")
         }
+        return pitch
     }
 
+    /// The map's current anchor, calculated after applying padding (if it exists)
     public var anchor: CGPoint {
-        get {
-            // TODO: Evaluate whether we should get the anchor from CameraView or not
-            return cameraView.anchor
-        } set {
-            cameraView.anchor = newValue
-        }
+
+        let paddding = padding
+        let xAfterPadding = center.x + paddding.left - paddding.right
+        let yAfterPadding = center.y + paddding.top - paddding.bottom
+        let anchor = CGPoint(x: xAfterPadding, y: yAfterPadding)
+
+        return anchor
     }
```

Comment:
I think we could remove all of these. Anchor could move somewhere else and be internal-only

Reply:
lets discuss this.

Reply:
Defer this decision to when CameraState lands